### DISCUSSION TITLE
DAKOTA for Inverse Uncertainty Quantification (IUQ)

### DISCUSSION THREAD

**Discussion by JohninSJ** [snl-dakota/dakota#90] [Q&A]
I am wondering if DAKOTA is capable for doing Inverse Uncertainty Quantification (IUQ). If yes, what methodologies are used, and where the application examples can be obtained. Could someone provide associated information? Much appreciated.

**Comment by briadam** [comment]
Could you please clarify what kind of inverse UQ you're seeking? If your goal is to characterize input parameter uncertainty to make a simulation model consistent with data presented to Dakota, it has (1) Bayesian methods (a couple MCMC methods and one experimental sampling-free method) to find a posterior distribution on the input parameters and (2) the ability to estimate hyper-parameters of named input probability distributions to maximize agreement with the data.

Dakota does not have distribution fitting methods for direct use on data, e.g., to determine the best-fit probability distribution and its parameters, given a dataset.

Perhaps once you clarify what's most helpful, this group can chime in more...

**Reply by JohninSJ** [reply]
Thank you very much for responses. We have model for predictions and experimental data, and would like to quantify the uncertainties of the predictions for selected parameters (as inputs to the model), by using the “Inverse Uncertainty Quantification (IUQ)” method. In the past, we did uncertainty quantification through “Forward Uncertainty Quantification (FUQ)” method. From some literatures associated with our professional field, Bayesian Inference is one of the methods we are interested in for IUQ.
